events/forms.py

```diff
@@ -89,13 +90,17 @@ def __init__(self, *args, **kwargs):
         if args:
             if 'state' in args[0]:
                 if args[0]['state'] and args[0]['state'] != '' and args[0]['state'] != 'None':
-                    choices = list(AcademicCenter.objects.filter(state_id = args[0]['state']).values_list('id', 'institution_name'))
+                    #choices = list(AcademicCenter.objects.filter(state_id = args[0]['state']).values_list('id', 'institution_name'))
+                    
+                    choices = list(AcademicCenter.objects.filter(state_id = args[0]['state']).annotate(combine=Concat('institution_name','academic_code')).values_list('id', 'combine'))
```

Comment:
1. The output of `value_list` is already a list so you might not want to have another `list()` function on top of it. 
2. You can just delete the comment above, keep a commented code just confuses everyone.